posthog/api/event_definition.py

```diff
@@ -9,7 +9,7 @@
 from posthog.api.utils import create_event_definitions_sql
 from posthog.constants import AvailableFeature, EventDefinitionType
 from posthog.exceptions import EnterpriseFeatureException
-from posthog.filters import TermSearchFilterBackend, term_search_filter_sql
+from posthog.filters import term_search_filter_sql
```

Comment:
Had a play myself and this indeed seems to do nothing. Could you also delete the `TermSearchFilterBackend` class and then we can get this in 💪 nice tidying!

Reply:
Removed in 9d54b4d